timemachine/fe/rbfe.py

```diff
@@ -151,6 +142,27 @@ class SimulationResult:
 # setup the initial state so we can (hopefully) bitwise recover the identical simulation
 # to help us debug errors.
 def setup_initial_states(st, host_config, temperature, lambda_schedule, seed):
+    """
+    Setup the initial states for a series of lambda values. It is assumed that the lambda schedule
+    is a monotonically increasing sequence in the closed interval [0,1].
+
+    Parameters
+    ----------
```

Comment:
nit: Doesn't mention the `st` parameter for the single topology object.

Reply:
willfix